src/tests/token/test_erc20.cairo

```diff
@@ -243,27 +230,27 @@ fn test__transfer_to_zero() {
 //
 
 #[test]
-#[available_gas(2000000)]
 fn test_transfer_from() {
     let mut state = setup();
     testing::set_caller_address(OWNER());
     state.approve(SPENDER(), VALUE);
     utils::drop_event(ZERO());
 
     testing::set_caller_address(SPENDER());
-    assert(state.transfer_from(OWNER(), RECIPIENT(), VALUE), 'Should return true');
+    assert!(state.transfer_from(OWNER(), RECIPIENT(), VALUE));
 
     assert_event_approval(OWNER(), SPENDER(), 0);
     assert_only_event_transfer(OWNER(), RECIPIENT(), VALUE);
 
-    assert(state.balance_of(RECIPIENT()) == VALUE, 'Should equal VALUE');
-    assert(state.balance_of(OWNER()) == SUPPLY - VALUE, 'Should equal SUPPLY - VALUE');
-    assert(state.allowance(OWNER(), SPENDER()) == 0, 'Should equal 0');
-    assert(state.total_supply() == SUPPLY, 'Total supply should not change');
+    let allowance = state.allowance(OWNER(), SPENDER());
+
+    assert_eq!(state.balance_of(RECIPIENT()), VALUE);
+    assert_eq!(state.balance_of(OWNER()), SUPPLY - VALUE);
+    assert_eq!(state.total_supply(), SUPPLY);
+    assert_eq!(allowance, 0);
```

Comment:
```suggestion
    let allowance = state.allowance(OWNER(), SPENDER());
    assert_eq!(allowance, 0);

    assert_eq!(state.balance_of(RECIPIENT()), VALUE);
    assert_eq!(state.balance_of(OWNER()), SUPPLY - VALUE);
    assert_eq!(state.total_supply(), SUPPLY);
```
+1 to making this a variable.
Though, I think it'd be better to keep the var next to the assertion since it's not used anywhere else